packages/harmony/src/components/layout/Box/Box.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from '@emotion/styled'
+
+import type { BoxProps } from './types'
+
+export const Box = styled.div(
+  ({
+    h,
+    w,
+    p,
+    ph,
+    pv,
+    pt,
+    pl,
+    pr,
+    pb,
+    m,
+    mh,
+    mv,
+    mt,
+    ml,
+    mr,
+    mb,
+    border,
+    borderRadius,
+    shadow
+  }: BoxProps) => {
+    const padT = pt ?? pv ?? p
+    const padB = pb ?? pv ?? p
+    const padL = pl ?? ph ?? p
+    const padR = pr ?? ph ?? p
+
+    const marginT = mt ?? mv ?? m
+    const marginB = mb ?? mv ?? m
+    const marginL = ml ?? mh ?? m
+    const marginR = mr ?? mh ?? m
```

Comment:
Personally I'm a bigger fan of X and Y instead of V and H (`px/py` vs `ph/pv`) but I'm happy either way

Reply:
I'll second that

Reply:
that's true. X/Y feel more obviously descriptive to me, but I'm sure I'll get used to H/V, so as JD said, I'm happy either way